exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/SqlConverter.java

```diff
@@ -478,7 +474,7 @@ public boolean isExplain() {
 
     @Override
     public boolean isExpand() {
-      return SqlToRelConverterConfig.DEFAULT.isExpand();
+      return false;
```

Comment:
Does this change affect expansion of subqueries in the WHERE clause only or also in the FROM clause ? If it does affect the latter, we would want to test it..